infoschema/tables.go

```diff
@@ -1457,6 +1459,27 @@ func GetStoreServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 	return servers, nil
 }
 
+// GetTiFlashStoreCount returns the count of tiflash server.
+func GetTiFlashStoreCount(ctx sessionctx.Context) (uint64, error) {
```

Comment:
```suggestion
func GetTiFlashStoreCount(ctx sessionctx.Context) (cnt uint64, err error) {
```
and remove the declaration below